app/browser/api/ledger.js

```diff
@@ -1882,15 +1882,33 @@ const onWalletProperties = (state, body) => {
     state = ledgerState.setInfoProp(state, 'currentRate', rate)
   }
 
+  // Grants
```

Comment:
Could this block perhaps be extracted to a separate function and exported? It shouldn't change any of the existing tests you've put in place.

Reply:
Not sure if needed